### PR TITLE
Bump to 1.19.2 and fix config save issue

### LINKED ISSUE #1: config does not copy
config does not copy to the server folder properly

### PR DESCRIPTION
**Pull request by Roosterseatyou**
Bumps to 1.19.2 with chat previewing supported as well as a config option to allow/disallow the chat preview to show. This also fixes #1 :)

### CODE REVIEW

**Comment by treecaptcha** [comment]
Everything here mostly seems good, one thing is that the uwuifier uses a random number generator and it does not guarantee that two runs of the same text will get the same output. One way to mitigate this would be to hash the string into a random seed through some fast hashing algorithm.

Also, we may want to look at the performance impact of getting the config value every time vs getting the value from a static variable as the static variable will probably be faster.

Thx for fixing #1 btw :)

\- TreeCaptcha (iCaffeine)

File: src/main/java/ml/treecaptcha/uwuify/Uwuify.java [hunk 1]
```diff
@@ -12,11 +12,15 @@ public void onEnable() {
...
     }
 
     @Override
     public void onDisable() {
         // Plugin shutdown logic
     }
+
+    public static final boolean USE_PREVIEW = uwu.getConfig().getBoolean("use-chat-preview");
+    public static final boolean SIGNS_UWUIFY = uwu.getConfig().getBoolean("signs-uwuify");
+    public static final boolean BOOKS_UWUIFY = uwu.getConfig().getBoolean("books-uwuify");
```
Comment (NoahvdAa) [inline src/main/java/ml/treecaptcha/uwuify/Uwuify.java:25]: This won't work, uwu is set after the static fields are initialized.

**Review by NoahvdAa — COMMENTED** [review]
The performance impact from getting it from the config should be minimal, iirc it's just a simple map lookup.

File: src/main/java/ml/treecaptcha/uwuify/Uwuify.java (hunk 1 above)
Comment (Roosterseatyou) [reply src/main/java/ml/treecaptcha/uwuify/Uwuify.java:25]: Crap you're right.

**Comment by treecaptcha** [comment]
seems good will merge after someone tests it :p
I'll make a release on merge too.

**Comment by Roosterseatyou** [comment]
All is working as intended!

**Comment by treecaptcha** [comment]
# Update to 1.19.2
- Add signs
- Add books
- Add support for chat preview
- Add some configuration 
- Add new uwuifiers that have the option to act the same each time they are run
- Only use chat preview if it is enabled in server.properties 
## fixes
- #1

**Pull request merged**